backend/app/views/spree/admin/style_guide/topics/typography/_colors.html.erb

```diff
@@ -0,0 +1,64 @@
+<%- snippet = capture do %>
+  <p>where the html example goes</p>
+<%- end %>
+
+<p>
+  The Solidus UI is made up mostly of a grey color palette with blue being used for interactive elements and red, green and yellow being used to give visual context to messaging.
+</p>
+
+<%-
+colors = [
```

Comment:
There is also this: https://viget.com/extend/sharing-data-between-sass-and-javascript-with-json